common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java

```diff
@@ -70,6 +70,11 @@ private Device() {
          */
         static final String PRODUCT_NAME = "MSAL.Android";
 
+        /**
+         * The String representing the sdk platform version.
+         */
+        static final String PRODUCT_VERSION = "0.2.2";
```

Comment:
Let's pull this from `BuildConfig` - this looks like something we've been forgetting to update if it's still @ `0.1.3`.

Reply:
Issue filed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/385

Reply:
Let's do that separately.